File: programmers/lv0/코딩기초트레이닝/조건문/주사위게임2/Solution.py
def solution(a, b, c):
    sum = a + b + c
    squareSum = a ** 2 + b ** 2 + c ** 2
    cubeSum = a ** 3 + b ** 3 + c ** 3
    if a != b and b != c and c != a:
        return sum
    elif a == b and b == c and c == a:
        return sum * squareSum * cubeSum
    else:
        return sum * squareSum

File: programmers/lv0/코딩기초트레이닝/조건문/주사위게임2/Solution.java
package lv0.코딩기초트레이닝.조건문.주사위게임2;

class Solution {
    public int solution(int a, int b, int c) {
        if (a != b && b != c && a != c) {
            return a + b + c;
        }
        else {
            int sum = a + b + c;
            int squareSum = a * a + b * b + c * c;
            int cubeSum = a * a * a + b * b * b + c * c * c;
            return a == b && b == c ? sum * squareSum * cubeSum : sum * squareSum;
        }
    }
}